src/molecules/TextLockup/text-lockups.module.scss

```diff
@@ -17,8 +13,73 @@ $subheader-color: $neutral-4;
   margin: 0 0 ru(.167);
 }
 
+.secondaryIcon, .smallScreenIcon {
+  position: absolute;
+  width: 100%;
+  transform: translate(-50%, -50%);
+  top: 50%;
+  left: 50%;
+}
```

Comment:
tried to get this to work just now but couldn't figure it out - I think I need to put a height on the icon to size it, which seems to prevent it from centering vertically